api-tester/src/main/java/com/revenuecat/apitester/java/StoreProductAPI.java

```diff
@@ -28,7 +28,9 @@ static void check(final StoreProduct product) {
         final String description = product.getDescription();
         final Period period = product.getPeriod();
         final String formattedPricePerMonth = product.formattedPricePerMonth(locale);
+        final Price pricePerWeek = product.pricePerWeek(locale);
         final Price pricePerMonth = product.pricePerMonth(locale);
+        final Price pricePerYear = product.pricePerYear(locale);
```

Comment:
We should also add versions without the optional parameter to these (I see we didn't have them for the `pricePerMonth` and `formattedPricePerMonth`...), same in the Kotlin API tests.